site/smart-accounts/modular-account/multisig-plugin/usage-guide.md

```diff
@@ -0,0 +1,128 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: Modular Account with Multisig Plugin • Getting started
+  - - meta
+    - name: description
+      content: Getting started with the Modular Account with Multisig Plugin in Account Kit
+  - - meta
+    - property: og:description
+      content: Getting started with the Modular Account with Multisig Plugin in Account Kit
+  - - meta
+    - name: twitter:title
+      content: Modular Account with Multisig Plugin • Getting started
+  - - meta
+    - name: twitter:description
+      content: Getting started with the Modular Account with Multisig Plugin in Account Kit
+---
+
+# Getting started with Modular Account with Multisig Plugin
+
+To start, follow the guide to set up Modular Account [here](../getting-started.md).
+
+## Modular Account with Multisig Plugin Actions
+
+### Creating a Account Client
+
+The first step is to create a Multisig Modular Account client 
+
+```ts
+import { LocalAccountSigner } from "@alchemy/aa-core";
+import { createMultisigAccountAlchemyClient } from "@alchemy/aa-alchemy";
+
+// Creating a 3/3 multisig account 
+const signers = [LocalAccountSigner.mnemonicToAccountSigner(
+    MODULAR_MULTISIG_ACCOUNT_OWNER_MNEMONIC,
+    { accountIndex: 0 }
+  ), LocalAccountSigner.mnemonicToAccountSigner(
+    MODULAR_MULTISIG_ACCOUNT_OWNER_MNEMONIC,
+    { accountIndex: 1 }
+  ), LocalAccountSigner.mnemonicToAccountSigner(
+    MODULAR_MULTISIG_ACCOUNT_OWNER_MNEMONIC,
+    { accountIndex: 2 }
+  )];
+
+const threshold = 3n;
+
+const owners = signers.map(async(s) => await s.getAddress())
```

Comment:
This won't work: you'll end up with an `Array<Promise<Address>>` instead of the `Array<Address>` you want. To wait on a list of promises, use `Promise.all`, e.g.
```ts
const owners = await Promise.all(
  signers.map((s) => s.getAddress()),
);
```